protocol/synthetix/storage.dump.sol

```diff
@@ -544,6 +544,8 @@ library Pool {
         uint64 __reserved1;
         uint64 __reserved2;
         uint64 __reserved3;
+        mapping(address => bool) disabledCollaterals;
+        mapping(address => uint256) IssuanceRatio;
```

Comment:
Strange that this is UpperCamelCase, maybe need to regenerate? Unsure if it matters.